sourcemash/categorize.py

```diff
@@ -0,0 +1,28 @@
+from collections import Counter
+
+def categorizeItem(title, text, category_dict):
+	cat1 = ""
+	cat2 = ""
+
+	categories = Counter()
+
+	title = title.split()
+	text = text.split()
+
+	# Get word counts from title, text of article
+	for source in [title, text]:
```

Comment:
This might be cleaner if you iterated over (title + text). Then you won't need the nested for loop on line 14.